test/unittest/base/test_jsonization.py

```diff
@@ -0,0 +1,70 @@
+# import superduperdb as s
+
+import typing as t
+
+import pydantic as pd
+import pytest
+
+IS_2 = pd.__version__.startswith('2')
+
+
+@pytest.mark.skipif(not IS_2, reason='test requires pydantic 2')
+def test_jsonization1():
```

Comment:
Nitpick: I prefer to have a bit more description about what the test is doing in the title or in comments if possible. It saves me time understanding what the test is actually doing when I inevitably break it in the future. :upside_down_face: (So, rather than having 1/2/3 as suffixes for the tests, maybe some comment on what distinguishes them.)